src/main/java/core/Main.java

```diff
@@ -0,0 +1,16 @@
+package core;
+
+import core.db.Storage;
+import core.model.Product;
+
+public class Main {
+    public static void main(String[] args) {
+        Product iphone = new Product("Iphone 11", 1200);
+        Product iphoneX = new Product("Iphone X", 1000);
+
+        Storage.addProduct(iphone);
+        Storage.addProduct(iphoneX);
+
+        Storage.products.forEach(System.out::println);
+    }
```

Comment:
why are you do not testing other methods (create, update and other)

and you miss some code from screen
https://mate-academy.github.io/jv-program-fulltime/02_jdbc_and_servlets/homework/content/new-injector.html
Use the injector as follows:

Reply:
thx, will be done